pkgs/applications/virtualization/toolbox/toolbox_paths.patch

```diff
@@ -0,0 +1,44 @@
+diff --git a/src/cmd/create.go b/src/cmd/create.go
+index 74e90b1..a95f4f0 100644
+--- a/src/cmd/create.go
++++ b/src/cmd/create.go
+@@ -207,6 +207,8 @@ func createContainer(container, image, release string, showCommandToEnter bool)
+ 	toolboxPathEnvArg := "TOOLBOX_PATH=" + toolboxPath
+ 	toolboxPathMountArg := toolboxPath + ":/usr/bin/toolbox:ro"
+ 
++	glibcMountArg := "@glibc@:@glibc@:ro"
++
+ 	var runtimeDirectory string
+ 	var xdgRuntimeDirEnv []string
+ 
+@@ -318,13 +320,15 @@ func createContainer(container, image, release string, showCommandToEnter bool)
+ 	var mntLink []string
+ 	var mntMount []string
+ 
+-	mntPath, _ := filepath.EvalSymlinks("/mnt")
+-	if mntPath == "/var/mnt" {
+-		logrus.Debug("/mnt is a symbolic link to /var/mnt")
+-		mntLink = []string{"--mnt-link"}
+-	} else {
+-		mntMount = []string{"--volume", "/mnt:/mnt:rslave"}
+-	}
++	if utils.PathExists("/mnt") {
++    mntPath, _ := filepath.EvalSymlinks("/mnt")
++    if mntPath == "/var/mnt" {
```

Comment:
Probably worthwhile trying to upstream this also.

Reply:
I can try, the other one closes an open non-nixos issue. This one is specific to nix.

Reply:
TBH I'm not sure we need this patch anymore

Reply:
https://github.com/containers/toolbox/pull/676